async/src/main/java/org/odk/collect/async/CoroutineAndWorkManagerScheduler.kt

```diff
@@ -37,7 +37,7 @@ class CoroutineAndWorkManagerScheduler(foregroundContext: CoroutineContext, back
             .setInputData(workManagerInputData)
             .build()
 
-        workManager.beginUniqueWork(tag, ExistingWorkPolicy.APPEND_OR_REPLACE, workRequest).enqueue()
+        workManager.beginUniqueWork(tag, ExistingWorkPolicy.REPLACE, workRequest).enqueue()
```

Comment:
From the `ExistingWorkPolicy.REPLACE docs`:

> If there is existing pending (uncompleted) work with the same unique name, cancel and delete it. Then, insert the newly-specified work.

This is the behaviour we want for auto send (currently the only thing that uses `networkDeferred`. We schedule auto send in two places:

1. When finalizing a form
2. When changing the auto send settings

For scenario 1, we could just use `KEEP` as we don't need to schedule another autosend task if one is already waiting to run (as the task submits all eligible forms). Scenario 2 requires `REPLACE` however as we might be changing the constraints of the task and so want to cancel the existing one.